calories/calories.rb

```diff
@@ -0,0 +1,84 @@
+require 'sqlite3'
+
+db = SQLite3::Database.new("daily_cal.db")
+db.results_as_hash = true
+
+continue = true
+$table_exists = false
+
+# allow user to add an item to the list.
+def add_item(db)
+	create_table_cmd = <<-SQL
+	CREATE TABLE IF NOT EXISTS items (
+		id INTEGER PRIMARY KEY,
+		name VARCHAR(255),
+		calories INT
+	)
+	SQL
+	db.execute(create_table_cmd)
+
+	$table_exists = true
+
+	puts "Please enter item name:"
+	item_name = gets.chomp
```

Comment:
Keep `gets.chomp` out of methods.  Use params instead.  `gets.chomp` is for driver code.